### DISCUSSION TITLE
Hello, is there a problem with editing mii foods?

### DISCUSSION THREAD

**Discussion by BGfnlraf** [Brionjv/Tomodachi-Life-Save-Editor#168] [Q&A]
I have been trying to change one of my miis favorite foods. When I go there, it says their favorite food is ice cream. This is happening to every other mii but the lookalike mii. I have tried to even go to past versions to edit this and it still doesn't work. Is this an issue for other people or just mii?

**Comment by blueturtel** [comment]
Same here! Changing the foods doesn't seem to change it permanently in the game.
